### PR TITLE
convert-doc: don't use doc if there was an error

### PR DESCRIPTION
**Pull request by jonathaningram**
Avoids panics.

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ func ConvertDoc(r io.Reader) (string, map[string]string, error) {
 		doc, err := mscfb.New(f)
 		if err != nil {
 			log.Printf("ConvertDoc: could not read doc: %v", err)
+			return
```
Comment (soulplant) [inline doc.go:39]: When we hit this case we will cause `ConvertDoc` to block forever, is that intentional? I suppose it's no worse than what happens now.

**Review by soulplant — DISMISSED** [review]

File: doc.go (hunk 1 above)
Comment (jonathaningram) [reply doc.go:39]: Yeah I wondered about it. I think it's the same as what is already happening:

- trying to use `doc` causes panic
- recover in the defer catches it and converts it to a log statement
- implied `return` happens just as my explicit `return` in this commit happens

Comment (jonathaningram) [reply doc.go:39]: I suppose my change is to just make it clearer the return was missed rather than fix underlying issues.

Comment (soulplant) [reply doc.go:39]: Yep, SGTM.

**Review by soulplant — APPROVED** [review]

**Pull request merged**